ui/components/multichain/account-overview/account-overview-tabs.tsx

```diff
@@ -0,0 +1,202 @@
+import React, { useContext } from 'react';
+import { useHistory } from 'react-router-dom';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import { ASSET_ROUTE } from '../../../helpers/constants/routes';
+import {
+  ///: BEGIN:ONLY_INCLUDE_IF(build-main)
+  SUPPORT_LINK,
+  ///: END:ONLY_INCLUDE_IF
+} from '../../../../shared/lib/ui-utils';
+import {
+  MetaMetricsEventCategory,
+  MetaMetricsEventName,
+} from '../../../../shared/constants/metametrics';
+import { MetaMetricsContext } from '../../../contexts/metametrics';
+import NftsTab from '../../app/nfts-tab';
+import AssetList from '../../app/asset-list';
+import TransactionList from '../../app/transaction-list';
+import { Tabs, Tab } from '../../ui/tabs';
+///: BEGIN:ONLY_INCLUDE_IF(build-main,build-mmi)
+import {
+  ///: END:ONLY_INCLUDE_IF
+  ///: BEGIN:ONLY_INCLUDE_IF(build-main)
+  Display,
+  ///: END:ONLY_INCLUDE_IF
+  ///: BEGIN:ONLY_INCLUDE_IF(build-main,build-mmi)
+  JustifyContent,
+} from '../../../helpers/constants/design-system';
+///: END:ONLY_INCLUDE_IF
+import {
+  Box,
+  ///: BEGIN:ONLY_INCLUDE_IF(build-main)
+  ButtonLink,
+  ButtonLinkSize,
+  IconName,
+  ///: END:ONLY_INCLUDE_IF
+} from '../../component-library';
+///: BEGIN:ONLY_INCLUDE_IF(build-mmi)
+import InstitutionalHomeFooter from '../../../pages/home/institutional/institutional-home-footer';
+///: END:ONLY_INCLUDE_IF
+import { AccountOverviewCommonProps } from './common';
+
+export type AccountOverviewTabsProps = AccountOverviewCommonProps & {
+  showTokens: boolean;
+  showNfts: boolean;
+  showActivity: boolean;
+};
+
+export const AccountOverviewTabs = ({
+  onTabClick,
+  ///: BEGIN:ONLY_INCLUDE_IF(build-main)
+  onSupportLinkClick,
+  ///: END:ONLY_INCLUDE_IF
+  defaultHomeActiveTabName,
+  showTokens,
+  showNfts,
+  showActivity,
+}: AccountOverviewTabsProps) => {
+  const history = useHistory();
+  const t = useI18nContext();
+  const trackEvent = useContext(MetaMetricsContext);
+
+  const tabPadding = 4;
+  const tabProps = {
+    activeClassName: 'account-overview__tab--active',
+    className: 'account-overview__tab',
+  };
```

Comment:
Let's use `useMemo` to ensure it retains the same reference across renders

```suggestion
  const tabProps = useMemo(() => ({
    activeClassName: 'account-overview__tab--active',
    className: 'account-overview__tab',
  }), []);
```